### ISSUE TITLE
loss

### ISSUE THREAD

**Issue by Jihnjockey** [Lornatang/SRGAN-PyTorch#69]

**Comment by Jihnjockey** [comment]
![image](https://user-images.githubusercontent.com/53632794/217734744-97cb1edb-2238-46e5-82e8-1d01d1a0fb16.png)
Why do you plus the pixel loss? I remembered that the author didn't add that loss part.

**Comment by Lingyun-Wang** [comment]
> ![image](https://user-images.githubusercontent.com/53632794/217734744-97cb1edb-2238-46e5-82e8-1d01d1a0fb16.png) Why do you plus the pixel loss? I remembered that the author didn't add that loss part.

I agree with you. In that paper, there are three kinds of SRGANs which are investigated in section 3.4: SRGAN-MSE (pixel-loss and adversarial-loss), SRGAN-VGG22(content-loss and adversarial-loss), and SRGAN-VGG54(content-loss and adversarial-loss).

**Comment by Lornatang** [comment]
For pixel loss， I research w/o pixle loss and w/ pixel loss, if you not increase pixel loss, 
the problem is abnormal color, difficult to converge... But I can't find the paper reproduced code by the original author, so at present, it is implemented according to the best loss method~~

**Comment by Lingyun-Wang** [comment]
> 

Thanks for your reply.

**Comment by Lingyun-Wang** [comment]
> For pixel loss， I research w/o pixle loss and w/ pixel loss, if you not increase pixel loss, the problem is abnormal color, difficult to converge... But I can't find the paper reproduced code by the original author, so at present, it is implemented according to the best loss method~~

Thanks for your reply.
